packages/core/src/__tests__/__e2e__/navigation-item/NavigationItem.cy.tsx

```diff
@@ -100,4 +100,88 @@ describe("GIVEN a NavItem", () => {
       });
     });
   });
+
+  describe("AND `render` is passed a render function", () => {
+    it("should call `render` to create parent item", () => {
+      const mockRender = cy.stub().as("render");
+      cy.mount(
+        <NavigationItem
+          active={true}
+          expanded={true}
+          href="https://www.saltdesignsystem.com"
+          level={2}
+          parent={true}
+          orientation="vertical"
+          render={mockRender}
+        >
+          Navigation Item
+        </NavigationItem>
+      );
+      cy.get("@render").should("have.been.calledWithMatch", {
```

Comment:
Is it possible to test that the element returned is actually rendered?

Reply:
done